lib/dialects/mysql/query-interface.js

```diff
@@ -1,89 +1,81 @@
 'use strict';
 
-/**
- Returns an object that treats MySQL's inabilities to do certain queries.
-
- @class QueryInterface
- @static
- @private
- */
-
 const sequelizeErrors = require('../../errors');
+const { QueryInterface } = require('../abstract/query-interface');
 
 /**
- A wrapper that fixes MySQL's inability to cleanly remove columns from existing tables if they have a foreign key constraint.
-
- @param  {QueryInterface} qi
- @param  {string} tableName     The name of the table.
- @param  {string} columnName    The name of the attribute that we want to remove.
- @param  {object} options
-
- @private
+ * Returns an object that treats MySQL's inabilities to do certain queries.
```

Comment:
Update documentation here, say this is the MySQL query interface. This comment will be exposed in documentation